source/_data/ui.yml

```diff
@@ -207,25 +207,25 @@
   ring: Adopt
   isNew: true
   description: "[Socket.IO](https://socket.io/) is a library that provides comprehensive support for WebSockets."
-  opinion: It is isomorphic (meaning we can use it on both the client & the server sides) and supports automatic reconnection and disconnection detection, making for a very robust and reliable WebSocket library. We use Socket.IO for realtime IPCs between many of our microservices.
+  opinion: It is isomorphic (meaning we can use it on both the client & the server sides) and supports automatic reconnection and disconnection detection, making for a very robust and reliable WebSocket library. We use Socket.IO for real-time IPCs between many of our microservices.
 - name: NPM
   quadrant: Testing & Build Tools
   ring: Adopt
   isNew: true
   description: "[NPM](https://www.npmjs.com/) is the default package manager that ships with Node.js."
-  opinion: For a number of projects, we simply use npm scripts for test runs & builds.
+  opinion: For a number of projects, we simply use npm scripts for test runs & build processes.
```

Comment:
The statement sounded to end abruptly. `test runs` is two-worded and `build processes` complement with that.